de1plus/updater.tcl

```diff
@@ -133,7 +135,7 @@ proc percent20encode {in} {
     set out $in
     regsub -all " " $out "%20" out
     #regsub -all "&" $out "%26" out
-    regsub -all {"} $out "%22" out
+    regsub -all "\"" $out "%22" out
```

Comment:
Why was this change necessary?

Reply:
It causes syntax validation problems in at least Tcl mode of emacs

No functional change, just a representational change of a string consisting of a the single "double quote" character